frontend/tests/blocks.spec.ts

```diff
@@ -51,160 +51,181 @@ for (const realmType of realms) {
- * changing certain parameters.
- */
-const testBlockEditing = async (page: Page, pos: number, blockType: Block["type"]) => {
-    await expect(page.getByRole("heading", { name: "Edit page" })).toBeVisible();
-
-    const saveButton = page.getByRole("button", { name: "Save" });
-    const editButton = page.getByLabel("Edit block").nth(pos);
-    await editButton.click();
-
-    switch (blockType) {
-        case "series": {
-            const newSeries = {
-                title: "Foxes are the very best!!",
-                description: "Cat software running on dog hardware.",
-            };
-            await test.step("Series can be changed", async () => {
-                const input = page.getByRole("combobox");
-                await input.pressSequentially(newSeries.title);
-                await page.getByRole("option", { name: newSeries.title }).click();
-                await saveButton.click();
-
-                await expect(page.getByRole("heading", { name: newSeries.title })).toBeVisible();
-            });
-
-            await test.step("Video order can be changed", async () => {
-                await editButton.click();
-                await page.getByLabel("Oldest first").setChecked(true);
-                await expect(page.getByLabel("Oldest first")).toBeChecked();
-                await saveButton.click();
-
-                await expect(page.getByRole("button", { name: "Choose video order" }))
-                    .toHaveText("Oldest first");
-            });
+                await test.step("Title block can be changed", async () => {
+                    const newTitle = "Nice test page";
+                    const editButton = page
+                        .getByRole("heading", { name: "Nice title" })
+                        .locator("..")
+                        .locator("..")
+                        .locator("..")
+                        .getByLabel("Edit block");
 
-            await test.step("Layout options can be changed", async () => {
-                await test.step("Title and videos (initial)", async () => {
-                    await expect(page.getByRole("heading", { name: newSeries.title }))
-                        .toBeVisible();
-                });
-
-                await test.step("Videos only", async () => {
                     await editButton.click();
-                    await page.getByLabel("Show title").setChecked(false);
-                    await page.getByLabel("Show description").setChecked(false);
+                    await page.getByPlaceholder("Title").fill(newTitle);
                     await saveButton.click();
 
-                    await expect(page.getByRole("heading", { name: newSeries.title }))
-                        .not.toBeVisible();
-                    await expect(page.getByText(newSeries.description))
-                        .not.toBeVisible();
+                    await expect(page.getByText(newTitle)).toBeVisible();
```

Comment:
Can this be `getByRole("heading", { name: newTitle })`. The locator as written is pretty generic. For example, if the save button doesn't do anything, the locator will probably just locate the text box (we just filled) and the test succeeds.